publish/dir.props

```diff
@@ -49,6 +49,5 @@
     <PublishRid Include="win-arm64" />
     <PublishRid Include="linux-arm" />
     <PublishRid Include="rhel.7-x64" />
```

Comment:
We just added these when we made the RPM packages.

/cc @vivmishra 

Honestly, I think this whole concept can go away now that we have the "publish/finalize" leg in a separate build step.  All this is used for is to make sure that all the other legs finished.  PipeBuild already does that for us.

/cc @chcosta